wled00/FX.cpp

```diff
@@ -4861,6 +4861,53 @@ static const char _data_FX_MODE_FLOWSTRIPE[] PROGMEM = "Flow Stripe@Hue speed,Ef
 #define XY(x,y) SEGMENT.XY(x,y)
 
 
+/////////////////////////////
+//    2D Analog Clock      //
+/////////////////////////////
+uint16_t mode_2DAnalogClock(void) {                  // By Andras Fekete (bandi13)
+  if (!strip.isMatrix || !SEGMENT.is2D()) return mode_static(); // not a 2D set-up
+
+  const int cols = SEGMENT.virtualWidth();
+  const int rows = SEGMENT.virtualHeight();
+  const int centerX = (cols-!(cols%2)) >> 1;  // use odd sized circle
+  const int centerY = (rows-!(rows%2)) >> 1;  // use odd sized circle
+  const int radius = min(centerX, centerY);
+  const bool soft = radius > 6 && SEGMENT.check2;
+
+  SEGMENT.fadeToBlackBy(240);
+  SEGMENT.drawCircle(centerX, centerY, radius, DARKGREY, soft);
```

Comment:
This ultimately makes a bright spot at the 3,6,9, and 12 o'clock positions. Just looks odd. Only seems to happen when brightness=10.

Reply:
I did not observe such behaviour on up to 20x20 (which I happen to have).

Consider changing segment size between 8x8 (non AA) and 32x32 (or larger if you have it). Then observe AA circle. Some pixels will be dimmer than others. And yes brightness plays a role when restoring/sampling a pixel using `getPixelColor()` if not using global buffer.

But I thought you wanted AA circle. Otherwise you should've used `fillCircle()` to get arbitrary sized border of uniform brightness. I did experiment with it but ultimately rejected it as it was obvious to me you wanted a smooth look.